### PR TITLE
Update CLI With Beta Apps

### PR DESCRIPTION
**Pull request by justin-shipstation**
Adding the ability to initialize a project using the new templates

`connect init --beta` will walk you through the same dialogue options as before but will now spit out working templates using the new connect-runtime & domain-api.

### CODE REVIEW

File: packages/connect/src/core/generators/beta-new.ts [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+      ...this.fs.readJSON("package.json", {}) as object,
+    };
+
+    const scopePresentInName = (name = ""): boolean => {
+      return !!name.match(/^@[a-z0-9-*~][a-z0-9-*._~]*/);
+    };
+
+    const defaults = {
+      name: capitalization.kebabCase(this.determineAppname(), { prefix: "shipengine" }),
+      scope: "@your-company-name",
```
Comment (jeversmann-shipstation) [inline packages/connect/src/core/generators/beta-new.ts:71]: I think @dougwile has been having trouble with these scopes making names too long, do we need to keep them?

File: packages/connect/src/core/generators/beta-new.ts [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+      type: "carrier",
+      version: "1.0.0",
+      author: this.githubUser
+        ? `${this.user.git.name()} @${this.githubUser}`
+        : this.user.git.name(),
+      dependencies: {},
+      ...this.pjson,
+      engines: {
+        node: ">=12.8.0",
```
Comment (jeversmann-shipstation) [inline packages/connect/src/core/generators/beta-new.ts:80]: Let's bump this to whatever 14 LTS is right now.

File: packages/connect/src/core/generators/beta-new.ts [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+        ...this.pjson.engines,
+      },
+      typescript: false,
```
Comment (jeversmann-shipstation) [inline packages/connect/src/core/generators/beta-new.ts:83]: What if we default this to `true` tho 🙊

**Review by jeversmann-shipstation — COMMENTED** [review]
The UUID generation in the demo metadata is cool, but do we know for sure that DM will handle those services (& etc) being published and then deleted? For instance, if I create a fresh app and publish the demo data immediately, and then create new, real data, will it get mad that the demo data is gone?

**Comment by justin-shipstation** [comment]
I believe data manager does the hard delete in that case if it can, but I would need to verify, this is mostly just a template to get them started so the hope would be instead of deleting they could just rename the service to the desired name

File: packages/connect/src/core/generators/beta-new.ts (hunk 1 above)
Comment (justin-shipstation) [reply packages/connect/src/core/generators/beta-new.ts:71]: Nope

File: packages/connect/src/core/generators/beta-new.ts (hunk 2 above)
Comment (justin-shipstation) [reply packages/connect/src/core/generators/beta-new.ts:80]: Kk

**Comment by justin-shipstation** [comment]
@jeversmann-shipstation I made the changes you suggested.

**Review by jeversmann-shipstation — APPROVED** [review]

**Comment by justin-shipstation** [comment]
increment minor

**Pull request merged**